### PR TITLE
doc: allow layers to ignore pAllocator

### PR DESCRIPTION
**Pull request by olvaffe**

### CODE REVIEW

File: loader/LoaderAndLayerInterface.md [hunk 1]
```diff
@@ -1115,6 +1115,10 @@ command to wrap the objects created by its lower layers, it must make sure its
 lower layers never see the wrapping objects, directly from itself or
 indirectly from its upper layers.
 
+When a layer requires host memory, it may ignore the provided allocators.  It
+is recommended to use the allocators when it is expected to be used as an
+implicit layer that is always enabled.
```
Comment (courtney-g) [inline loader/LoaderAndLayerInterface.md]: How about:
... Layers should use allocators if the layer is intended to run in a production environment, such as an implicit layer that is always enabled. That will allow applications to include the layer's memory usage.

**Comment by olvaffe** [comment]
That is much better.  Updated.

**Comment by ianelliottus** [comment]
+1.  LGTM

**Comment by tobine** [comment]
LGTM :+1:
Did this get Khronos workgroup agreement? Can we close #371?

**Comment by courtney-g** [comment]
We discussed during the WSI call. We should leave this open and make it an agenda topic for next Tuesday. We can then review the findings of the WSI group, get agreement from the rest of the WG and then be done.

**Comment by jon-lunarg** [comment]
committed to master and closing #371

**Pull request closed without merging**